packages/notifi-react-card/lib/hooks/useNotifiSubscribe.ts

```diff
@@ -183,13 +183,8 @@ export const useNotifiSubscribe: () => Readonly<{
     const finalTelegramId = inputTelegramId === '' ? null : inputTelegramId;
 
     let finalPhoneNumber = null;
-    if (inputPhoneNumber !== '') {
-      if (inputPhoneNumber.startsWith('+')) {
-        finalPhoneNumber = inputPhoneNumber;
-      } else {
-        // Assume US for now
-        finalPhoneNumber = `+1${inputPhoneNumber}`;
-      }
+    if (isValidPhoneNumber(inputPhoneNumber)) {
+      finalPhoneNumber = inputPhoneNumber;
```

Comment:
do we no longer need the `+` logic?

Reply:
yeah, no longer need it.
[
    "+1",
    "+61",
    "+43",
    "+32",
    "+55",
    "+45",
    "+358",
    "+33",
    "+49",
    "+852",
    "+36",
    "+354",
    "+60",
    "+47",
    "+63",
    "+48",
    "+351",
    "+65",
    "+82",
    "+34",
    "+46",
    "+41",
    "+886",
    "+44"
]